Classes/Exception/GlossaryCanNotCreatedException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WebVision\WvDeepltranslate\Exception;
+
+class GlossaryCanNotCreatedException extends \Exception
```

Comment:
* Naming: I'd suggest to use `FailedToCreateGlossaryException` as name, or at at least add a `be` ( GlossaryCanNotBeCreatedException ).
* No need to open a exception extension change here, let's make the exception final.

Reply:
I have made your suggestions for changes.